librad/src/git/fetch.rs

```diff
@@ -115,6 +115,42 @@ pub mod refspecs {
         ]
     }
 
+    pub fn peek_of<P, R>(urn: &Urn<R>, remote_peer: P) -> Vec<Fetchspec>
```

Comment:
Ah yes.

Maybe we can collapse this into one method which takes a parameter `local_mapping: impl Into<Option<PeerId>>` or something? Also, how do you feel about adjusting the (pure) tests as well?

Reply:
Ya, was going to see what you thought of the placement of the refs before cleaning up :smile: